src/System.Management.Automation/FormatAndOutput/common/StringDecorated.cs

```diff
@@ -98,7 +98,7 @@ private string PlainText
         }
 
         // graphics/color mode ESC[1;2;...m
-        private const string GraphicsRegex = @"(\x1b\[\d+(;\d+)*m)";
+        private const string GraphicsRegex = @"(\x1b\[\d*(;\d+)*m)";
```

Comment:
This allows a string like `esc[;2m` -- is this considered a valid escape sequence?

Reply:
/cc @mklement0

Reply:
Testing with zsh and iTerm2, `ESC[;35m` renders as purple, so it seems to be accepted